Watchman.DomainModel/Configuration/MappedConfiguration.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Watchman.DomainModel.Configuration
+{
+    public abstract class MappedConfiguration<T> : IMappedConfiguration
+    {
+        public abstract T Value { get; set; }
+        public ulong ServerId { get; }
+        public string Name { get; }
+
+        public MappedConfiguration(ulong serverId)
+        {
+            this.Name = this.GetType().Name;
+            this.ServerId = serverId;
+        }
+
+        public override bool Equals(object obj)
```

Comment:
jeszcze możesz nadpisać operator ==
```csharp
public static bool operator ==(IMappedConfiguration<T> obj1, IMappedConfiguration<T> obj2)
```
ale nie jestem pewny, może equals robi to samo 
ładniej wtedy wygląda 

a jeszcze lepiej jest stworzyć takie coś jak tutaj
https://dev.azure.com/Squader/Squader/_git/SquaderBackend?path=%2FSquader%2FSquader.Domain%2FValueObject.cs

i każdy obiekt który dziedziczy po ValueObject ma do tego dostęp

Reply:
z tym że tam są refleksje => czyli działa to wolno 
jak jest masa obiektów i odpalalibyśmy to szybko
to nie jest to najlepszy sposób

tzn to wolne działanie pokazuje sie przy dużej skali dopiero 
bo przy małych przykładach to będzie niemożliwe do zauważenia przez użytkownika